player/local_window_player_movement.py

```diff
@@ -0,0 +1,42 @@
+import pygame
+
+from player import *
+
+SCREEN_WIDTH = 1280
+SCREEN_HEIGHT = 720
+
+FPS = 60
+
+
+class LocalWindowPlayerMovement:
+    def __init__(self, width, height):
+        pygame.init()
+
+        self.width = width
+        self.height = height
+        self.screen = pygame.display.set_mode((self.width, self.height))
+        self.clock = pygame.time.Clock()
+        self.background_color = (144, 201, 120)
+        self.is_running = True
+
+    def tick(self):
+        self.clock.tick(FPS)
```

Comment:
Please make TODO to delete this method when we start to blend all together because then it should not be there

Reply:
Yes I agree with that